graphbolt/src/csc_sampling_graph.cc

```diff
@@ -199,7 +307,7 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::SampleNeighborsImpl(
                   local_grain_size);
 
               for (scalar_t i = begin; i < end; ++i) {
-                const auto nid = nodes[i].item<int64_t>();
+                const auto nid = *nodes[i].data_ptr<int64_t>();
```

Comment:
const auto nid = nodes.data_ptr<int64_t>()[i];
It is best if the dataptr is assigned outside of the loop.